src/app.ts

```diff
@@ -44,29 +44,34 @@ import {
 
 const FSH_VERSION = '2.0.0';
 
-app().catch(e => {
+function logUnexpectedError(e: Error) {
   logger.error(`SUSHI encountered the following unexpected error: ${e.message}`);
   process.exit(1);
-});
+}
+app().catch(logUnexpectedError);
```

Comment:
@jafeltra - super nitpick, can you put a blank line after the function?

Reply:
No problem @jafeltra! See [19c7aa7](https://github.com/FHIR/sushi/pull/1163/commits/19c7aa78bf985da030dea19726aebd67edbbd399).